### ISSUE TITLE
Incorrect error due to `ready` task state when spawn the container

### ISSUE THREAD

**Issue by yacchin1205** [jupyterhub/dockerspawner#372]
### Bug description
The error `RuntimeError: Service jupyter-xxxx not running: {'State': 'ready'}` sometimes occurs when launching the container.

#### Expected behaviour
The container is launched without any error.

#### Actual behaviour
The actual error message:

```
[E 2020-05-27 13:09:22.593 JupyterHub user:626] Unhandled error starting xxxx's server: Service jupyter-xxxx not running: {'Message'
: 'prepared',
     'PortStatus': {},
     'State': 'ready',
     'Timestamp': '2020-05-27T13:09:22.581302293Z'}
ERROR:asyncio:Task exception was never retrieved
future: <Task finished coro=<BaseHandler.spawn_single_user() done, defined at /opt/conda/lib/python3.6/site-packages/jupyterhub/handlers/base.py:697> exception=RuntimeError
("Service jupyter-xxxx not running: {'Message': 'prepared',\n 'PortStatus': {},\n 'State': 'ready',\n 'Timestamp': '2020-05-27T13:09:22.581302293Z'}",)>
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/site-packages/jupyterhub/handlers/base.py", line 889, in spawn_single_user
    timedelta(seconds=self.slow_spawn_timeout), finish_spawn_future
  File "/opt/conda/lib/python3.6/site-packages/jupyterhub/handlers/base.py", line 807, in finish_user_spawn
    await spawn_future
  File "/opt/conda/lib/python3.6/site-packages/jupyterhub/user.py", line 642, in spawn
    raise e
  File "/opt/conda/lib/python3.6/site-packages/jupyterhub/user.py", line 546, in spawn
    url = await gen.with_timeout(timedelta(seconds=spawner.start_timeout), f)
  File "/opt/conda/lib/python3.6/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/opt/conda/lib/python3.6/site-packages/dockerspawner/dockerspawner.py", line 1022, in start
    yield self.start_object()
  File "/opt/conda/lib/python3.6/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/opt/conda/lib/python3.6/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.send(value)
  File "/opt/conda/lib/python3.6/site-packages/dockerspawner/swarmspawner.py", line 266, in start_object
    "Service %s not running: %s" % (self.service_name, pformat(status))
RuntimeError: Service jupyter-xxxx not running: {'Message': 'prepared',
 'PortStatus': {},
 'State': 'ready',
 'Timestamp': '2020-05-27T13:09:22.581302293Z'}
```

### How to reproduce

Perform the steps below repeatedly.

1. Open Control Panel
2. Stop My Server
3. Start My Server

### Your personal set up

 - OS: Ubuntu 18.04
 - JupyterHub Version: 1.0.0
 - Configuration: SwarmSpawner

**Comment by yacchin1205** [comment]
I assume that `READY` state does not appear in the document of Docker Swarm https://docs.docker.com/engine/swarm/how-swarm-mode-works/swarm-task-states/ but an issue https://github.com/docker/docker.github.io/issues/7317 indicates actually `READY` state exists...

So I will try to add "ready" to https://github.com/jupyterhub/dockerspawner/blob/master/dockerspawner/swarmspawner.py#L257 😃

**Comment by minrk** [comment]
Fixed by #396
